CONTRIBUTING.md

```diff
@@ -615,7 +615,7 @@ that should hopefully be there, to a new section named `UPGRADE FROM 42.3.1 to 4
 
 #### Upgrading code comments and deprecation messages
 
-All occurences of `42.x` in comments or deprecation messages should be updating
+All occurrences of `42.x` in comments or deprecation messages should be updating
```

Comment:
@pgrimaud can you please run your bot on https://github.com/sonata-project/dev-kit ? This file is copied from that repository, so your changes will be overwritten on the next copy unless the typo is fixed there too.

Reply:
Yes, sorry my bot is a little bit dumb actually... But it's learning about different workflow.

It submit another PR here : https://github.com/sonata-project/dev-kit/pull/663 😄